spec/mailers/user_mailer_spec.rb

```diff
@@ -1,11 +1,71 @@
 require 'rails_helper'
 
-RSpec.describe DeviseMailer, type: :mailer do
-  describe 'confirmation mails' do
+RSpec.describe Devise.mailer, type: :mailer do
+
```

Comment:
Space — the final frontier. These are the voyages of the starship `develop`. Its ongoing mission — to program strange new apps — to seek out new smells and new optimizations — to boldly code where no man has coded before.